README.md

```diff
@@ -112,6 +114,145 @@ plugins: [
 ];
 ```
 
+## Running Babel on the generated code
+
+You can run rollup-plugin-babel on the output files instead of the input files by using `babel.generated(...)`. This can be used to perform code transformations on the resulting chunks and is the only way to transform Rollup's auto-generated code. By default, the plugin will be applied to all outputs:
+
+```js
+// rollup.config.js
+import babel from 'rollup-plugin-babel';
+export default {
+	input: 'main.js',
+	plugins: [babel.generated()],
+	output: [
+		{ file: 'bundle.cjs.js', format: 'cjs' },
+		{ file: 'bundle.esm.js', format: 'esm' },
+	],
+};
+```
+
+If you only want to apply it to specific outputs, you can use it as an output plugin:
+
+```js
+// rollup.config.js
+import babel from 'rollup-plugin-babel';
+export default {
+	input: 'main.js',
+	output: [
+		{ file: 'bundle.js', format: 'esm' },
+		{
+			file: 'bundle.es5.js',
+			format: 'esm',
+			plugins: [babel.generated({ presets: ['@babel/env'] })],
+		},
+	],
+};
+```
+
+The `include`, `exclude` and `extensions` options are ignored when the when using `.generated()` will produce warnings, and there are a few more points to note that users should be aware of.
+
+You can also run the plugin twice on the code, once when processing the input files to transpile special syntax and once on the output to transpile to a lower compatibility target:
+
+```js
+// rollup.config.js
+import babel from 'rollup-plugin-babel';
+export default {
+	input: 'main.js',
+	plugins: [babel({ presets: ['@babel/preset-react'] })],
+	output: [
+		{
+			file: 'bundle.js',
+			format: 'esm',
+			plugins: [babel.generated({ presets: ['@babel/env'] })],
+		},
+	],
+};
+```
+
+### Finding Babel configuration options
+
+Besides Babel options that are passed directly to this plugin, this plugin will only respect project-wide [Babel configuration files](https://babeljs.io/docs/en/config-files) based on the current working directory when applying it to the generated code.
```

Comment:
I know we are here in subsection and all, but rather than mentioning "this plugin" in "this plugin will only respect project-wide" I would make it somehow more explicit that this is just about the `.generated` variant.